app/views/appointments/_list.html.erb

```diff
@@ -3,23 +3,23 @@
     <div class="list-element">
       <div class="left-content">
         <p>
-          <a>Psychologist:</a>
+          <a><%= t('psychologist') %>:</a>
           <% @psychologist.each do |p|%>
              <%= p.name if a.psychologist_id == p.id%> 
           <% end %> <br>
-          <a>Date: </a> <%= a.datetime %> <br>
-          <a>Price: </a> $50 usd
+          <a><%= t('date') %>: </a> <%= a.datetime %> <br>
+          <a><%= t('price') %>: </a> $23 usd
```

Comment:
Regarding the amount of 23 dollars, will there be an option to also implement it in the national currency when changing languages? (In the case of Mexico in pesos)